listener/subgraph_definition/dfusion

```diff
@@ -7,7 +7,7 @@ dataSources:
   - kind: ethereum/contract
     name: SnappBaseEvents
     source:
-      address: '0x9561C133DD8580860B6b7E504bC5Aa500f0f06a7'
+      address: '0xe982E462b094850F12AF94d21D470e21bE9D0E9C'
```

Comment:
Weird, why did the address change?

Reply:
Because of https://github.com/gnosis/dex-contracts/pull/198